witness/src/Dscp/Witness/Web/Logic.hs

```diff
@@ -234,3 +239,13 @@ checkFairCV =
         { fairCVCheckResults = results
         , fairCVFullyValid = all and results
         }
+
+checkFairCVPDF :: forall ctx m. WitnessWorkMode ctx m => ByteString -> m FairCVAndCheckResult
+checkFairCVPDF pdf = do
+    let maybeFairCV = do
+            fairCVencoded <- Pdf.project (Pdf.MaxSearchLength (Just 2048)) (Pdf.PDFBody pdf)
+            Aeson.decode $ BS.fromStrict fairCVencoded
+
+    fairCV   <- maybe (throwM InvalidFormat) pure maybeFairCV
```

Comment:
One more minor thing: if I use this endpoint and get this `InvalidFormat` which usually stands for wrong request body or query parameters format, that would be misleading because I did everything correctly and it's PDF which is wrong (and it is made not by me but by some another endpoint).
Maybe creating a separate exception would be better, something like "invalid certificate pdf format".